pyproject.toml

```diff
@@ -81,7 +85,12 @@ pydocstyle.convention = "google"
 "zeus/optimizer/perseus/common.py" = ["N805"]
 "zeus/optimizer/perseus/server/router.py" = ["B008"]
 "zeus/util/pydantic_v1.py" = ["F403"]
+"zeus/optimizer/batch_size/**/commands.py" = ["N805"]
+"zeus/optimizer/batch_size/**/models.py" = ["N805"]
+"zeus/optimizer/batch_size/server/config.py" = ["N805"]
+"zeus/optimizer/batch_size/server/router.py" = ["B008"]
+"zeus/optimizer/batch_size/common.py" = ["N805"]
 "zeus/device/gpu.py" = ["N802", "N803"]
 
-[tool.pytest.ini_options]
-addopts = "--numprocesses auto"
+#[tool.pytest.ini_options]
+#addopts = "--numprocesses auto"
```

Comment:
?

Reply:
Hmm, I am having an issue with parallel testing. I have a fixture that sets up the DB and cleans up after the testing is done. Seems like parallel processing causes this process to overlap. Any thoughts?

Reply:
Does some of the stuff here help? https://github.com/pytest-dev/pytest-xdist/issues/84